text/0080-datetime-extension.md

```diff
@@ -442,15 +356,12 @@ forbid(
 }
 ```
 
-## Potential extensions
-
-### Ext. A: Provide a `currentTime` function
-
-The examples above expect `currentTime` to be passed in the context during the authorization request. An alternative would be to provide a `currentTime()` extension function that computes the current time at policy evaluation time. This has a drawback, when testing policies, in that the evaluator must be somehow extended to mock the time.
-
 ## Unresolved questions
 
 ### Should we consider local time at all?
 
 Writing programs that involve dates and time is tricky. Standardizing on UTC representations for all dates and times trades achieving evaluator correctness more quickly at the expense of decreased readability for policy authors and auditors.
 
+### Should we provide a `currentTime` function?
+
+The examples above expect `currentTime` (or `now`) to be passed in the context during the authorization request. An alternative would be to provide a `currentTime()` extension function that computes the current time at policy evaluation time. This has a drawback, when testing policies, in that the evaluator must be somehow extended to mock the time.
```

Comment:
The other draw back is that leap seconds might be observable during an evaluation. We’re skirting around that by saying “time is constant during evaluation, so leap seconds is a system problem.”